app/src/main/java/org/feature/fox/coffee_counter/ui/profile/ProfileView.kt

```diff
@@ -143,23 +149,47 @@ fun ProfileIcon(viewModel: IProfileViewModel) {
             .fillMaxWidth(),
         horizontalAlignment = Alignment.CenterHorizontally,
     ) {
-        Image(
-            painter = painter,
-            contentDescription = stringResource(R.string.profile_image_label),
-            modifier = Modifier
-                .wrapContentSize()
-                .size(150.dp)
-                .clickable(
-                    enabled = true,
-                    onClickLabel = "Clickable profile image",
-                    onClick = {
-                        launcher.launch("image/*")
-                    }
+        Box(
+            contentAlignment = Alignment.BottomEnd,
+        ) {
+            Image(
+                painter = painter,
+                contentDescription = stringResource(R.string.profile_image_label),
+                modifier = Modifier
+                    .wrapContentSize()
+                    .size(150.dp)
+                    .clickable(
+                        enabled = true,
+                        onClickLabel = "Clickable profile image",
```

Comment:
change placeholder :D

Reply:
Should be resolved in [8d5872d](https://github.com/Morphclue/code-camp-2/pull/112/commits/8d5872df0479abe0b3ea5f619a4183337e0bf5f9).